tests/bluetooth/tester/src/btp_mesh.c

```diff
@@ -5163,218 +5154,137 @@ static const struct btp_handler handlers[] = {
 		.func = proxy_identity_enable,
 	},
 #if defined(CONFIG_BT_MESH_PROXY_CLIENT)
-	{
-		.opcode = BTP_MESH_PROXY_CONNECT,
-		.expect_len = sizeof(struct btp_proxy_connect_cmd),
-		.func = proxy_connect
-	},
+	{.opcode = BTP_MESH_PROXY_CONNECT,
+	 .expect_len = sizeof(struct btp_proxy_connect_cmd),
+	 .func = proxy_connect},
 #endif
 #if defined(CONFIG_BT_MESH_SAR_CFG_CLI)
-	{
-		.opcode = BTP_MESH_SAR_TRANSMITTER_GET,
-		.expect_len = sizeof(struct btp_mesh_sar_transmitter_get_cmd),
-		.func = sar_transmitter_get
-	},
-	{
-		.opcode = BTP_MESH_SAR_TRANSMITTER_SET,
-		.expect_len = sizeof(struct btp_mesh_sar_transmitter_set_cmd),
-		.func = sar_transmitter_set
-	},
-	{
-		.opcode = BTP_MESH_SAR_RECEIVER_GET,
-		.expect_len = sizeof(struct btp_mesh_sar_receiver_get_cmd),
-		.func = sar_receiver_get
-	},
-	{
-		.opcode = BTP_MESH_SAR_RECEIVER_SET,
-		.expect_len = sizeof(struct btp_mesh_sar_receiver_set_cmd),
-		.func = sar_receiver_set
-	},
+	{.opcode = BTP_MESH_SAR_TRANSMITTER_GET,
```

Comment:
Did you run `clang-format` on these commits? This change and some others don't seem necessary. Did CI complained about these lines?

Reply:
Yes, I started running the "./scripts/ci/check_compliance.py -c 1471c2f81430fd0a381c37d263a4d0ff966e0cb4..HEAD" locally at the end. And fixed all the faults that it complained on, or I would have fixed just a few for every push I did.